internal/cmd/connect/command_cluster_delete.go

```diff
@@ -88,5 +77,19 @@ func (c *clusterCommand) validateArgs(cmd *cobra.Command, environmentId, kafkaCl
 		return err
 	}
 
-	return connectorIdToName, deletion.ValidateArgsForDeletion(cmd, args, resource.Connector, describeFunc)
+	if err := deletion.ValidateArgsForDeletion(cmd, args, resource.Connector, describeFunc); err != nil {
+		return err
+	}
+
+	if len(args) == 1 {
+		if err := form.ConfirmDeletionWithString(cmd, resource.Connector, args[0], connectorIdToName[args[0]]); err != nil {
```

Comment:
How about getting the name from the describe API call instead of passing this `map` in?

Reply:
I need these names in the main `delete` function since `V2Client.DeleteConnector` takes the name instead of the id, so I pass in the map to avoid making two of every describe call.

I'm just recycling the map for this line.